quilt/README.md

```diff
@@ -0,0 +1,241 @@
+```
+
+## Step 3: Run `ray-runner.spec`
+
+### Start the Quilt daemon
+Execute `quilt daemon` in one terminal session. This will start the quilt
+process (and hang). Do not close this session. Open a new terminal session
+(<kbd>Ctrl</kbd> + <kbd>T</kbd> or <kbd>&#8984;</kbd> + <kbd>T</kbd> on OS X).
+
+### Run `ray-runner.spec`
+We suggest you read through [`ray.spec`](ray.spec) and
+[`ray-runner.spec`](ray-runner.spec) to get a feel of how the infrastructure is
+defined.
+
+While in the `path/to/amplab/ray/quilt/` directory, execute `quilt run
+ray-runner.spec`. Quilt will set up `nWorker + 1` VMs. You will see logs pop up
+on the terminal session where you executed `quilt daemon`.
+
+### Accessing the VM
+It will take a while for the VMs to boot up, for Quilt to configure the network,
+and for Docker containers to be initialized. When the logs in the Quilt daemon
+display machines with IP addresess and say `Connected`:
+
+```
+quilt daemon
+INFO [Aug 16 15:04:45.104] db.Machine:
+    Machine-2{Master, Amazon us-west-1 m4.large, Disk=32GB}
+    Machine-3{Worker, Amazon us-west-1 m4.large, Disk=32GB}
+INFO [Aug 16 15:04:45.105] db.Cluster:
+    Cluster-1{vivian-ray-1, ACL: [136.152.142.34/32]}
+INFO [Aug 16 15:04:56.364] Attempt to boot machines.                count=2
+INFO [Aug 16 15:05:59.867] Successfully booted machines.
+INFO [Aug 16 15:06:10.772] db.Machine:
+    Machine-2{Master, Amazon us-west-1 m4.large, sir-042pcgha, PublicIP=54.193.20.57, PrivateIP=172.31.11.175, Disk=32GB}
+    Machine-3{Worker, Amazon us-west-1 m4.large, sir-042r83pp, PublicIP=54.183.84.139, PrivateIP=172.31.9.27, Disk=32GB}
+INFO [Aug 16 15:08:15.114] db.Machine:
+    Machine-2{Master, Amazon us-west-1 m4.large, sir-042pcgha, PublicIP=54.193.20.57, PrivateIP=172.31.11.175, Disk=32GB}
+    Machine-3{Worker, Amazon us-west-1 m4.large, sir-042r83pp, PublicIP=54.183.84.139, PrivateIP=172.31.9.27, Disk=32GB, Connected}
+INFO [Aug 16 15:09:45.116] db.Machine:
+    Machine-2{Master, Amazon us-west-1 m4.large, sir-042pcgha, PublicIP=54.193.20.57, PrivateIP=172.31.11.175, Disk=32GB, Connected}
+    Machine-3{Worker, Amazon us-west-1 m4.large, sir-042r83pp, PublicIP=54.183.84.139, PrivateIP=172.31.9.27, Disk=32GB, Connected}
+```
+
+You can ssh into a machine by executing `quilt ssh <MACHINE_ID>` in the same
+terminal session you executed. To view available machines, execute `quilt
+machines`, which will give output like this:
+
+```
+quilt machines
+Machine-2{Master, Amazon us-west-1 m4.large, sir-042pcgha, PublicIP=54.193.20.57, PrivateIP=172.31.11.175, Disk=32GB, Connected}
+Machine-3{Worker, Amazon us-west-1 m4.large, sir-042r83pp, PublicIP=54.183.84.139, PrivateIP=172.31.9.27, Disk=32GB, Connected}
+```
+
+The `MACHINE_ID` is the number after `Machine-` in the output of `quilt
+machines`. For example, to ssh into in the Master VM, execute `quilt ssh 2`.
+
+### Finding Ray
+Once you are in a ssh session with the Master VM, execute `swarm ps`. This will
```

Comment:
Sigh.  Let's hold off merging this until Kevin's latest PR is in, cleans this bit up a lot.